src/one_dragon/base/operation/application/application_factory_manager.py

```
@@ -0,0 +1,456 @@
+            if init_file.exists():
+                init_spec = importlib.util.spec_from_file_location(
+                    pkg_module_name,
+                    init_file,
+                    submodule_search_locations=[str(pkg_dir)]
+                )
+                if init_spec and init_spec.loader:
+                    init_module = importlib.util.module_from_spec(init_spec)
+                    sys.modules[pkg_module_name] = init_module
+                    init_spec.loader.exec_module(init_module)
+            else:
+                # 无 __init__.py 时创建命名空间包，确保 dotted import 正常工作
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)